Makefile

```diff
@@ -30,6 +30,13 @@ ifndef BUILD_NUMBER
 	endif
 endif
 
+ifndef DOCKER_IMAGE_NAME
+	DOCKER_IMAGE_NAME = nebula
+endif
+ifndef DOCKER_IMAGE_TAG
+	DOCKER_IMAGE_TAG = lates
```

Comment:
Should this be `DOCKER_IMAGE_TAG = latest` instead?

Reply:
It sure should, thanks for that!